vendor/vendor.json

```diff
@@ -146,10 +138,20 @@
 			"revisionTime": "2019-06-13T14:39:42Z"
 		},
 		{
-			"checksumSHA1": "z6OakgyT4iR/xAKnnko9jXdJPlI=",
+			"checksumSHA1": "3HWurU2V4/n3BjF7UpJu132zGrk=",
+			"origin": "github.com/mendersoftware/cli",
 			"path": "github.com/urfave/cli",
```

Comment:
I would replace the path to `github.com/mendersoftware/cli` as the packages differ.

Reply:
Do you mean modifying then all the `import` statements as well? I don't know what the recommended practice is here. Or package is just a fork, so I thought the best would be to respect original path.